### PR TITLE
Added test for Rack adapter using Rack::Test to ensure we do not depend on non-Rack env vars.

### PR DESCRIPTION
**Pull request by bethesque**

### CODE REVIEW

**Comment by bethesque** [comment]
@Asmod4n, can you review when you have a moment? Travis is green.

**Comment by bethesque** [comment]
As a note of explanation, this is the failing test I added to prove that using REQUEST_URI did not work in all situations.

**Comment by Asmod4n** [comment]
Merge mere ☺

File: spec/webmachine/adapters/rack_spec.rb [hunk 1]
```diff
@@ -34,3 +35,23 @@
     end
   end
 end
+
+describe Webmachine::Adapters::Rack do
+  let(:app) do
+    Webmachine::Application.new.tap do |app|
```
Comment (seancribbs) [inline spec/webmachine/adapters/rack_spec.rb]: `tap` is [not necessary](https://github.com/seancribbs/webmachine-ruby/blob/master/lib/webmachine/application.rb#L48).

**Comment by bethesque** [comment]
Thanks @seancribbs, removed.

**Comment by seancribbs** [comment]
:+1:

**Comment by bethesque** [comment]
Travis CI is failing, but for the usual reasons (have inspected each failure!), not because of this change, so I will merge.

**Pull request merged**